api/src/main/java/jakarta/data/metamodel/Expression.java

```diff
@@ -39,20 +39,20 @@ default Restriction<T> equalTo(Expression<? super T,V> expression) {
         return BasicRestriction.of(this, EqualTo.expression(expression));
     }
 
-    default Restriction<T> in(Set<V> values) {
+    default Restriction<T> in(Collection<V> values) {
         if (values == null || values.isEmpty())
             throw new IllegalArgumentException("values are required");
 
         return BasicRestriction.of(this, In.values(values));
     }
 
-    default Restriction<T> in(
-            @SuppressWarnings("unchecked") V... values) {
+    @SuppressWarnings("unchecked")
+    default Restriction<T> in(V... values) {
         return BasicRestriction.of(this, In.values(values));
     }
 
-    default Restriction<T> in(
-            @SuppressWarnings("unchecked") Expression<? super T,V>... expressions) {
+    @SuppressWarnings("unchecked")
+    default Restriction<T> in(Expression<? super T,V>... expressions) {
```

Comment:
This `@SuppressWarnings` annotation was in the wrong place, at least according to _my_ version of `javac` and IntelliJ. Apparently there's been some movement here, because `@SuppressWarnings` didn't used to be enough to get rid of this useless warning. I'm very happy they're _finally_ provided a way to make it go away!

Reply:
As stated in the other PR, moving it to the method gets rid of complaints in the build output, so this is a good update.